python/kwiver/sprokit/pipeline/process.cxx

```diff
@@ -6,7 +6,7 @@
 typedef std::set<std::string> string_set; // This has to be done first thing, or the macro breaks
 PYBIND11_MAKE_OPAQUE(string_set)
 
-#include <python/kwiver/vital/util/pybind11.h>
+#include <pybind11/pybind11.h>
```

Comment:
Nit: Move down into the same block as the other pybind11 includes. Other instances of this edit could also use movement down into their respective pybind include blocks.

Side question: Are the other includes necessary in addition to this top-level pybind include?